tools/gn

```diff
@@ -656,6 +655,8 @@ def to_gn_wasm_args(args, gn_args):
   # than forced to true, once the recipes are updated.
   # https://github.com/flutter/flutter/issues/113303
   gn_args['build_canvaskit'] = True
+  # TODO(mdebbar): Read this from an input argument.
```

Comment:
Is there an GitHub issue we can link to this TODO here?

Reply:
Done